### PR TITLE
Add glacier in creatkeebs folder

### PR DESCRIPTION
**Pull request by Timliuzhaolu**
## Description
Add glacier in creatkeebs folder

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/creatkeebs/glacier/config.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
+
+/* prevent stuck modifiers */
+#define PREVENT_STUCK_MODIFIERS
+
+
```
Comment (zvecr) [inline keyboards/creatkeebs/glacier/config.h:63]:
```suggestion
```

File: keyboards/creatkeebs/glacier/config.h [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#define DIODE_DIRECTION COL2ROW
+
+/* number of backlight levels */
+
+#ifdef BACKLIGHT_PIN
+#define BACKLIGHT_LEVELS 0
+#endif
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
```
Comment (zvecr) [inline keyboards/creatkeebs/glacier/config.h:47]:
```suggestion
#define DEBOUNCE 5
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/creatkeebs/glacier/config.h [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/creatkeebs/glacier/config.h:56]: Could you explicitly define which animations are enabled? 
```suggestion
#define RGBLIGHT_EFFECT_BREATHING
#define RGBLIGHT_EFFECT_RAINBOW_MOOD
#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
#define RGBLIGHT_EFFECT_SNAKE
#define RGBLIGHT_EFFECT_KNIGHT
#define RGBLIGHT_EFFECT_CHRISTMAS
#define RGBLIGHT_EFFECT_STATIC_GRADIENT
#define RGBLIGHT_EFFECT_RGB_TEST
#define RGBLIGHT_EFFECT_ALTERNATING
#define RGBLIGHT_EFFECT_TWINKLE
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by Timliuzhaolu** [comment]
Hi @drashna am I missing any requested changes? It has been half a month since the last time I committed...

File: keyboards/creatkeebs/glacier/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# glacier
+A 80% keyboard
+
+![glacier](https://cdn.shopify.com/s/files/1/0481/5840/8853/products/4b8d52820aab652152cd749123d3f8b_6a073c5e-336d-47b4-90fc-8b0fe5a790fe_720x.png?v=1641814324)
```
Comment (fauxpark) [inline keyboards/creatkeebs/glacier/readme.md:4]: Could you please reupload this to imgur?

File: keyboards/creatkeebs/glacier/info.json [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+                {"x":7.5, "y":0},
+                {"x":8.5, "y":0},
+                {"x":9.75, "y":0},
+                {"x":10.75, "y":0},
+                {"x":11.75, "y":0},
+                {"x":12.75, "y":0},
+                {"x":14, "y":0},
+                {"x":15.25, "y":0},
+		{"x":16.25, "y":0},
+		{"x":17.25, "y":0},
```
Comment (fauxpark) [inline keyboards/creatkeebs/glacier/info.json:24]:
```suggestion
                {"x":16.25, "y":0},
                {"x":17.25, "y":0},
```

File: keyboards/creatkeebs/glacier/info.json [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+                {"x":7, "y":1.25},
+                {"x":8, "y":1.25},
+                {"x":9, "y":1.25},
+                {"x":10, "y":1.25},
+                {"x":11, "y":1.25},
+                {"x":12, "y":1.25},
+                {"x":13, "y":1.25, "w":2},
+                {"x":15.25, "y":1.25},
+		{"x":16.25, "y":1.25},
+		{"x":17.25, "y":1.25},
```
Comment (fauxpark) [inline keyboards/creatkeebs/glacier/info.json:41]:
```suggestion
                {"x":16.25, "y":1.25},
                {"x":17.25, "y":1.25},
```

File: keyboards/creatkeebs/glacier/info.json [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+                {"x":7.5, "y":2.25},
+                {"x":8.5, "y":2.25},
+                {"x":9.5, "y":2.25},
+                {"x":10.5, "y":2.25},
+                {"x":11.5, "y":2.25},
+                {"x":12.5, "y":2.25},
+                {"x":13.5, "y":2.25, "w":1.5},
+                {"x":15.25, "y":2.25},
+		{"x":16.25, "y":2.25},
+		{"x":17.25, "y":2.25},
```
Comment (fauxpark) [inline keyboards/creatkeebs/glacier/info.json:58]:
```suggestion
                {"x":16.25, "y":2.25},
                {"x":17.25, "y":2.25},
```

File: keyboards/creatkeebs/glacier/info.json [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+                {"x":0, "y":5.25, "w":1.5},
+                {"x":1.5, "y":5.25},
+                {"x":2.5, "y":5.25, "w":1.5},
+                {"x":4, "y":5.25, "w":7},
+                {"x":11, "y":5.25, "w":1.5},
+                {"x":12.5, "y":5.25},
+		{"x":13.5, "y":5.25, "w":1.5},
+                {"x":15.25, "y":5.25},
+		{"x":16.25, "y":5.25},
+		{"x":17.25, "y":5.25}
```
Comment (fauxpark) [inline keyboards/creatkeebs/glacier/info.json:94]:
```suggestion
                {"x":13.5, "y":5.25, "w":1.5},
                {"x":15.25, "y":5.25},
                {"x":16.25, "y":5.25},
                {"x":17.25, "y":5.25}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by Timliuzhaolu** [comment]
Thanks again guys! Shall we wait for the other reviewer who requested changes a while ago to approve or is it good to go

**Comment by Timliuzhaolu** [comment]
> Thanks again guys! Shall we wait for the other reviewer who requested changes a while ago to approve or is it good to go

Sorry for the pinning but not sure about whether you guys saw the previous message... @fauxpark @drashna

**Comment by drashna** [comment]
Thanks!

**Pull request merged**